rust/arrow/src/array.rs

```diff
@@ -1659,13 +1710,80 @@ mod tests {
         field_types.push(Field::new("a", DataType::Boolean, false));
         field_types.push(Field::new("b", DataType::Int64, false));
         let struct_array_data = ArrayData::builder(DataType::Struct(field_types))
+            .len(4)
+            .add_child_data(boolean_data.clone())
+            .add_child_data(int_data.clone())
+            .build();
+        let struct_array = StructArray::from(struct_array_data);
+
+        assert_eq!(boolean_data, struct_array.column(0).data());
+        assert_eq!(int_data, struct_array.column(1).data());
+    }
+
+    #[test]
+    fn test_struct_array_slice() {
+        let mut null_bits: [u8; 1] = [0; 1];
+        bit_util::set_bit(&mut null_bits, 0);
+        bit_util::set_bit(&mut null_bits, 2);
+        bit_util::set_bit(&mut null_bits, 4);
```

Comment:
nit: assuming that this evaluates to `10101`, do you mind changing it to something like `11101` or something where there isn't a repeating pattern? Checking the 3 ordered values of `101--` and `--101` would still result in a test passing without us knowing if it read the correct 3 bit masks.

Reply:
Good suggestion. Will change.